optionals/compat_cup_recruit/script_component.hpp

```diff
@@ -0,0 +1,17 @@
+#define COMPONENT compat_cup_recruit
```

Comment:
```suggestion
#define COMPONENT compat_cup_weapons
```
And make it general CUP Weapons compatibility component for all of TAC Mods (like we have for Vehicles).